contracts/strategies/ltip-simple/LTIPSimpleStrategy.sol

```diff
@@ -0,0 +1,666 @@
+    }
+
+    /// ===============================
+    /// ======= External/Custom =======
+    /// ===============================
+
+    /// @notice Review recipient(s) application(s)
+    /// @dev You can review multiple recipients at once or just one. This can only be called by a pool manager and
+    ///      only during active registration.
+    /// @param __recipientIds Ids of the recipients
+    /// @param _recipientStatuses Statuses of the recipients
+    function reviewRecipients(address[] calldata __recipientIds, Status[] calldata _recipientStatuses)
+        external
+        virtual
+        onlyPoolManager(msg.sender)
+        onlyActiveReview
+    {
+        // make sure the arrays are the same length
+        uint256 recipientLength = __recipientIds.length;
+        if (recipientLength != _recipientStatuses.length) revert INVALID();
+
+        for (uint256 i; i < recipientLength;) {
+            Status recipientStatus = _recipientStatuses[i];
+            address recipientId = __recipientIds[i];
+            Recipient storage recipient = _recipients[recipientId];
+
+            // only pending applications can be updated
+            // and the new status can only be Accepted or Rejected
+            if (
+                recipient.recipientStatus != Status.Pending
+                    || (recipientStatus != Status.Accepted && recipientStatus != Status.Rejected)
+            ) {
+                revert RECIPIENT_ERROR(recipientId);
+            }
+            recipient.recipientStatus = recipientStatus;
+
+            emit Reviewed(recipientId, recipientStatus, msg.sender);
+
+            unchecked {
+                ++i;
+            }
+        }
+    }
+
+    /// @notice Cancel (remove) recipient(s) application(s)
+    /// @dev You can remove multiple recipients at once or just one.
+    /// @param __recipientIds Ids of the recipients
+    function cancelRecipients(address[] calldata __recipientIds) external virtual onlyPoolManager(msg.sender) {
+        uint256 recipientLength = __recipientIds.length;
+
+        for (uint256 i; i < recipientLength;) {
+            address recipientId = __recipientIds[i];
+            Recipient storage recipient = _recipients[recipientId];
+
+            // if the status is none or appealed then revert
+            if (recipient.recipientStatus == Status.None || recipient.recipientStatus == Status.Canceled) {
+                revert RECIPIENT_ERROR(recipientId);
+            }
+
+            recipient.recipientStatus = Status.Canceled;
+
+            emit Canceled(recipientId, msg.sender);
+
+            unchecked {
+                ++i;
+            }
+        }
+    }
+
+    /// @notice Toggle the status between active and inactive.
+    /// @dev 'msg.sender' must be a pool manager to close the pool. Emits a 'PoolActive()' event.
+    /// @param _flag The flag to set the pool to active or inactive
+    function setPoolActive(bool _flag) external onlyPoolManager(msg.sender) {
```

Comment:
Any chance this can be derived using the timestamps we have on init ? or is this easier ?

Reply:
For reference from our conversation: The idea is to be able to have perpetual rounds and this enables -temporarily- stopping the round.